src/i18n/fr_FR.ts

```diff
@@ -1145,89 +736,86 @@ export default {
     },
     buildingIcons: [
         'fire',
-        'rss',
-        'hospital',
-        'clinic-medical',
         'graduation-cap',
-        'shield-alt',
+        'ambulance'
+        '',

```

Comment:
You can find the right icons at [https://fontawesome.com/icons?d=gallery&m=free](https://fontawesome.com/icons?d=gallery&m=free)